tools/data_generators/rbac_dummy_data_generator.py

```diff
@@ -40,10 +40,13 @@
 # awx
 from awx.main.models import (  # noqa
     Credential, CredentialType, Group, Host, Inventory, Job, JobEvent,
-    JobHostSummary, JobTemplate, Label, Organization, PrimordialModel, Project,
+    JobHostSummary, JobTemplate, Label, Organization, Project,
```

Comment:
Instead of this diff, @kialam would you mind trying this:

```diff
diff --git a/awx/main/models/__init__.py b/awx/main/models/__init__.py
index 80b8c2fd2..2e4821dcf 100644
--- a/awx/main/models/__init__.py
+++ b/awx/main/models/__init__.py
@@ -7,7 +7,7 @@ from django.db.models.signals import pre_delete  # noqa

 # AWX
 from awx.main.models.base import (  # noqa
-    BaseModel, prevent_search, CLOUD_INVENTORY_SOURCES, VERBOSITY_CHOICES
+    BaseModel, PrimordialModel, prevent_search, CLOUD_INVENTORY_SOURCES, VERBOSITY_CHOICES
 )
```

Reply:
👍